cmd/libguestfs/entrypoint.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash -xe
 
-LIBGUESTFS_PATH=${LIBGUESTFS_PATH:=/tmp/guestfs}
-LIBGUEST_APPLIANCE=/usr/local/lib/guestfs/downloaded
-mkdir -p ${LIBGUESTFS_PATH}
-tar -Jxf ${LIBGUEST_APPLIANCE} -C ${LIBGUESTFS_PATH} --strip-components=1
+LIBGUESTFS_PATH=${LIBGUESTFS_PATH:=/usr/local/lib/guestfs/appliance}
 
-touch ${LIBGUESTFS_PATH}/done
+# Keep done file for backward compatibility.
+# Certain setups or the tests might use the done file to understand when the appliance has been extracted. Create the file if it doesn't exist, for example if LIBGUESTFS_PATH isn't the default one.
+if [ ! -f ${LIBGUESTFS_PATH}/done  ]; then
+  touch ${LIBGUESTFS_PATH}/done
+fi
```

Comment:
What could be overriding `LIBGUESTFS_PATH` to make it point somewhere else? AFAICT the value comes from `pkg/virtctl/guestfs/guestfs.go` and is completely hardcoded.

The container image this script is used as the entrypoint of is built by us anyway, and bazel decides where the files go, which is in `/usr/local/lib/guestfs/appliance`. If you wanted the files to live somewhere else for whatever reason, you'd have to patch the build system and virtctl anyway. So I don't understand why this check would be needed.

Reply:
Running manually the image for example, or used by another tools you could overwrite the `LIBGUESTFS_PATH` environmental variable. But I might overthink

Reply:
If you overwrite the `LIBGUESTFS_PATH`, you also don't find the appliance anymore but maybe the user create a symlink to `/usr/local/lib/guestfs` and use another path

Reply:
I don't really see any legitimate use case for that. You'd only make things more complicated for no gain.

If you are not okay with the default path you will need to build your own images, patch the build system and virtctl. If you've gone that far, you might as well patch the entrypoint script too.

I think you could basically get rid of the entrypoint script at this point. Or reduce it to just

```bash
#!/bin/bash -xe

bash
```

But if you prefer to keep the check right now and potentially clean up in a follow-up PR, that's fine by me as well :)

Reply:
Yes, you are right let's keep it simple